tools/topology/topology2/cavs/cavs-sdw.conf

```diff
@@ -51,6 +51,11 @@ Define {
 	DMIC0_PCM_CAPS 'Passthrough Capture 13'
 	DMIC0_PIPELINE_STREAM_NAME 'copier.DMIC.14.1'
 	USE_CHAIN_DMA	"false"
+	DEEP_BUFFER_PIPELINE_ID         15
+	DEEP_BUFFER_PCM_ID              13
```

Comment:
why can't we use the same PCM IDs are for IPC3/topology1?

Reply:
it should be 31. I am a little number blind